src/gestures/use-focus-gesture.ts

```diff
@@ -0,0 +1,90 @@
+import { TargetAndTransition } from "../types"
+import { VisualElement } from "../render/VisualElement"
+import { AnimationType } from "../render/VisualElement/utils/animation-state"
+import { VariantLabels } from "../motion/types"
+import { useDomEvent } from "../events/use-dom-event"
+
+/**
+ * @public
+ */
+export interface FocusHandlers {
+    /**
+     * Properties or variant label to animate to while the focus gesture is recognised.
+     *
+     * @library
```

Comment:
In Framer, a `Frame` is always a `div`, so it'll never receive a focus event. So you can safely remove lines 14 to 21

Reply:
Resolved 👍